contrib/experimental_dict_builders/benchmarkDictBuilder/benchmark.c

```diff
@@ -344,10 +352,9 @@ int main(int argCount, const char* argv[])
     memset(&coverParam, 0, sizeof(coverParam));
     coverParam.zParams = zParams;
     coverParam.splitPoint = 1.0;
-    coverParam.d = d;
     coverParam.steps = 40;
     coverParam.nbThreads = 1;
-    const int coverOptResult = benchmarkDictBuilder(srcInfo, maxDictSize, NULL, &coverParam, NULL);
+    const int coverOptResult = benchmarkDictBuilder(srcInfo, maxDictSize, NULL, &coverParam, NULL, NULL);
```

Comment:
`ZDICT_optimizeTrainFromBuffer_cover()` will fill `coverParam` with the selected params. We can use the parameters to run it a second time with a fixed `k` and `d`, to get the speed when it doesn't optimize. We can also use that `k` value to run `FASTCOVER` without having to optimize.